azure-mgmt-trafficmanager/src/main/java/com/microsoft/azure/management/trafficmanager/TrafficManagerProfiles.java

```diff
@@ -66,4 +67,10 @@
      * @return a handle to cancel the request
      */
     ServiceFuture<CheckProfileDnsNameAvailabilityResult> checkDnsNameAvailabilityAsync(String dnsNameLabel, ServiceCallback<CheckProfileDnsNameAvailabilityResult> callback);
+
+    /**
+     * @return @return the default Geographic Hierarchy used by the Geographic traffic routing method.
```

Comment:
I don't think "Geographic Hierarchy` is a proper nous, so no need for the capitalization